triviagame.html

```diff
@@ -2,48 +2,65 @@
 1. Choose your trivia category: movies, songs, or history.
 2. Answer 4 numerical questions
 3. You will get hints to whether your guess is too high or too low
-4. You have 3 guesses each to get the answer right
-5. (Optional) You have 10 seconds each before your time runs out! -->
+4. !! You have 3 guesses each to get the answer right
+5. !! Add a quit feature to exit the screen at any prompt
+6. !! You have 10 seconds each before your time runs out! 
+7. !! Create objects to manage the trivia and answers. -->
 
 <title>Adam's Trivia Game!</title>
 
 <script>
+
+	var movieQuestions = [
+		"How many films has Christian Bale been in?",
```

Comment:
Would you like us to correct this one, or leave it as is? I think we've
pretty much all set our sublime profile defaults to space-indents and
2-space tabs at this point.

On Tue, Jan 13, 2015 at 7:47 PM, Mel Quark <notifications@github.com> wrote:

> In triviagame.html
> <https://github.com/adamstelle/movietrivia/pull/2#discussion-diff-22915282>
> :
>
> >
> >  <title>Adam's Trivia Game!</title>
> >
> >  <script>
> > +
> > +	var movieQuestions = [
> > +		"How many films has Christian Bale been in?",
>
> use two-space, soft-tabs (for readability on github), please =)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adamstelle/movietrivia/pull/2/files#r22915282>.
>